src/EventListener/AddContentLanguageHeaderListener.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\HttpCache\EventListener;
+
+use eZ\Publish\API\Repository\Values\Content\Content;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+final class AddContentLanguageHeaderListener implements EventSubscriberInterface
+{
+    public const CONTENT_LANGUAGE_HEADER = 'xlang';
```

Comment:
Shouldn't it be rather `X-Lang` or `x-lang`? It's rather a general convention to separate `X` header with a dash. Seems Ibexa DXP also respects that.

Reply:
As it is strongly related to `xkey` header - which is, well, without `-` - I've taken a naming conversion from there. No strong opinion though.

Reply:
> As it is strongly related to `xkey` header - which is, well, without `-` - I've taken a naming conversion from there. No strong opinion though.

Hmm, Maybe xkey has some other special meaning, but:
```
$ curl -i https://127.0.0.1:8000/
HTTP/2 200 
cache-control: max-age=0, must-revalidate, private, s-maxage=86400
content-type: text/html; charset=UTF-8
date: Tue, 18 Jan 2022 12:20:16 GMT
expires: Tue, 18 Jan 2022 12:20:16 GMT
set-cookie: eZSESSID98defd6ee70dfb1dea416cecdf391f58=kd9g2s55f7ddj9h9c56btjd85q; path=/; secure; httponly; samesite=lax
vary: X-User-Context-Hash
vary: X-Editorial-Mode
x-cache-debug: 1
x-cache-tags: ez-all,siso_user_10,siso_menu,c52,l2,siso_pagelayout_footer,content-101,content-102,content-104,content-103,ct42,pl1,p1,p2
x-debug-token: abcdef
x-debug-token-link: https://127.0.0.1:8000/_profiler/abcdef
x-powered-by: Ibexa Commerce v4
x-robots-tag: noindex
```
in this context, it makes `xlang` inconsistent